openshift_cli_installer/utils/gcp_utils.py

```diff
@@ -0,0 +1,103 @@
+import json
+import os
+import shutil
+from pathlib import Path
+import tempfile
+from pyaml_env import parse_config
+
+from simple_logger.logger import get_logger
+from openshift_cli_installer.utils.const import GCP_STR
+
+LOGGER = get_logger(name=__name__)
+TMP_FILE_NAME = tempfile.NamedTemporaryFile(suffix="installer.json").name
+
+
+def get_gcp_params():
+    gcp_params = {}
```

Comment:
no need to declare empty dict of `gcp_params`
Just build the dict with all the necessary keys,vals and return it